### PR TITLE
Modify ImportError traceback for _aligners

### PR DESCRIPTION
**Pull request by chris-rands**
This pull request relates to an issue raised on the mailing list [here](http://mailman.open-bio.org/pipermail/biopython/2018-June/016485.html). I also saw the same question asked on Stack overflow [here](https://stackoverflow.com/questions/51296034/importerror-cannot-import-name-aligners-biopython), and I then later independently encountered the issue myself because I had used some soft links rather than doing a proper install. The error can occur on a few imports, for example:

```
>>> from Bio import SeqIO
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/path/biopython/Bio/SeqIO/__init__.py", line 375, in <module>
    from Bio.Align import MultipleSeqAlignment
  File "path/biopython/Bio/Align/__init__.py", line 22, in <module>
    from Bio.Align import _aligners
ImportError: cannot import name '_aligners'
```

I found the error slightly cryptic until I read through the mailing list thread, and perhaps I am not alone. Michiel noted in the mailing list thread [here](http://mailman.open-bio.org/pipermail/biopython/2018-June/016492.html) that `numpy` have a more informative traceback, I suggest modifying the traceback in this case in a similar way:

```
>>> from Bio import SeqIO
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/path/biopython/Bio/SeqIO/__init__.py", line 375, in <module>
    from Bio.Align import MultipleSeqAlignment
  File "/path/biopython/Bio/Align/__init__.py", line 29, in <module>
    raise new_exc
ImportError: cannot import name '_aligners': you should not import directly from the
                         biopython source directory; please exit the source
                         tree and re-launch your code from there
```

Thoughts?

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Bio/Align/__init__.py [hunk 1]
```diff
@@ -19,7 +19,14 @@
 from Bio.SeqRecord import SeqRecord, _RestrictedDict
 from Bio import Alphabet
 
-from Bio.Align import _aligners
+try:
+    from Bio.Align import _aligners
+except ImportError as e:
+    new_exc = ImportError("""{}: you should not import directly from the
```
Comment (peterjc) [inline Bio/Align/__init__.py]: Don't use a triple quote string, which inserts the new lines breaks as is. Use single quotes (which will auto-concatenate)

**Comment by peterjc** [comment]
TravisCI failure linked to ReportLab, see #1737 1737

**Comment by peterjc** [comment]
TravisCI failure caused by #1737 (latest ReportLab release).

**Comment by mdehoon** [comment]
Looks good to me.

As this error typically appears when a user runs "python setup.py install" and directly afterwards tries to import Bio, some day we should reorganize the directory structure such that Bio/ etc. is not a direct subdirectory of biopython, then this error is unlikely to happen at all.

**Pull request merged**